packages/react-charting/docs/TestPlans/MultiStackedBarChart/componentTests.md

```diff
@@ -0,0 +1,55 @@
+**Multi stacked bar Chart – Component test plan**
```

Comment:
Add a test case to show placeholder bar.

Reply:
done